afs/afs-mapdb-storage/src/main/java/com/powsybl/afs/mapdb/storage/MapDbAppStorage.java

```diff
@@ -306,12 +309,16 @@ public void setDescription(String nodeId, String description) {
         NodeInfo nodeInfo = getNodeInfo(nodeId);
         nodeInfo.setDescription(description);
         nodeInfoMap.put(nodeUuid, nodeInfo);
+        pushEvent(new NodeDescriptionUpdated(nodeId, description),
+                String.valueOf(NodeEventType.NODE_DESCRIPTION_UPDATED));
```

Comment:
General comment. For events coming from storage I would use 'STO§RAGE" as unique topic.

Reply:
But i think creating a topic per event type is more interesting for events consumption to handle them separately.

Reply:
@sylvlecl what do you think  ?

Reply:
I think it depends on how topics will be used.
Will listeners be associated with one topic ? with several topics ?
We could also provide a `Set<String> STORAGE_TOPICS` to discriminate between storage events and others.

Reply:
For now, I think we should keep a topic per event type for two reasons :
1) to not saturate queues in the messaging queue platform so enhancing performance.
2) to consume the event that we want to consume from the suitable queue and not consume all  events and then filter them. 
Then it depends on the messaging queue platform. consuming message from multiple queues in not hard to implement if the queuing platform is RabbitMQ,  but i think that's a harder with kafka platform. i'm working on that to determine what is the better solution.